supportedBackends/bcs/src/main/scala/cromwell/backend/impl/bcs/BcsJob.scala

```diff
@@ -23,7 +22,7 @@ final case class BcsJob(name: String,
                   runtime: BcsRuntimeAttributes,
                   stdoutPath: Option[Path],
                   stderrPath: Option[Path],
-                  batchCompute: BatchComputeClient) {
+                  backend: BcsAsyncBackendJobExecutionActor) {
```

Comment:
Why was this switched from just the `batchCompute` to the `backend.bcsClient`? I'm sure this isn't the only case in the codebase, but calling methods directly on Actors feels weird to me, even if it is from a utility class like this.

https://stackoverflow.com/questions/13291346/call-methods-on-akka-actors-in-scala#answer-13293167

Reply:
Actually, this is left by the previous discarded credentials refresh solution. 
Thanks!